composer.json

```diff
@@ -66,7 +66,7 @@
         "behat/mink-extension": "^2.3",
         "behat/mink-goutte-driver": "^1.2",
         "behatch/contexts": "^3.2",
-        "coduo/php-matcher": "^3.1.2",
+        "coduo/php-matcher": "dev-patch-2@dev",
```

Comment:
Not a blocking issue, but is this `coduo/php-matcher` robust? I think this is the second or third time we've had to use it as non-stable, right?

Reply:
Because I've made another PR to them ;)